### PR TITLE
WIP: Collection miscellaneous updates for 1.13.0 release.

### PR DESCRIPTION
**Pull request by charris**
Work in progress, may merge with another cleanup PR before release. Just keeping stuff here so I don't forget.

### CODE REVIEW

File: numpy/linalg/linalg.py [hunk 1]
```diff
@@ -1528,7 +1528,8 @@ def matrix_rank(M, tol=None):
     Parameters
     ----------
     M : {(M,), (..., M, N)} array_like
-        input vector or stack of matrices
+        input vector or stack of matrices. The ability handle stacked matrices
```
Comment (eric-wieser) [inline numpy/linalg/linalg.py]: Missing `to`

**Comment by eric-wieser** [comment]
Sorry about that. Also, I forgot to add to release notes in #8827 - I could combine these and the notes there into one, since they're basically the same thing

File: numpy/linalg/linalg.py (hunk 1 above)
Comment (charris) [inline numpy/linalg/linalg.py]: Fixed. Thanks.

**Comment by charris** [comment]
Sure, go ahead.

**Comment by eric-wieser** [comment]
Updated that PR

**Pull request closed without merging**